specification/v4.0/OSDM-online-api-v4.0.0-draft.yml

```diff
@@ -13997,7 +13997,7 @@ components:
       properties:
         departureTime:
           description: |
-            Needs to in local date time format.
+            Needs to in local date time format. If departure time is provided the departure time must be omitted.
```

Comment:
Should be
"If departure time is provided the arrival time must be omitted." to mimic line 14006